homeassistant/components/mqtt/valve.py

```diff
@@ -302,65 +298,63 @@ def _process_position_valve_update(
             return
         self._update_state(state)
 
+    @callback
+    def _state_message_received(self, msg: ReceiveMessage) -> None:
+        """Handle new MQTT state messages."""
+        payload = self._value_template(msg.payload)
+        payload_dict: Any = None
+        position_payload: Any = payload
+        state_payload: Any = payload
+
```

Comment:
This typing doesn't look correct, but its existing code. Something to be cleaned up later